src/core/StelSkyImageTile.cpp

```diff
@@ -469,7 +469,8 @@ void StelSkyImageTile::loadFromQVariantMap(const QVariantMap& map)
 	{
 		const QVariant& polyRaDec = polyList.at(i);
 		QVector<Vec3d> vertices;
-		for (const auto& vRaDec : polyRaDec.toList())
```

Comment:
What warnings were emitted here?

Reply:
The clangd analysis inside QtCreator often complains in those loop constructs about (IIRC, there are two usual such messages, and I am tired) temporary structures detaching. Better have no warnings. Setting the list const seems to fix that. Weird enough I cannot reproduce this one right now.

Reply:
I don't find any way this would actually lead to problems, because the temporary bound to the range is life-extended, see [here](https://en.cppreference.com/w/cpp/language/range-for#Temporary_range_expression). Similarly for the same case below.